document/travis-deploy.sh

```diff
@@ -13,13 +13,18 @@ function doCompile {
   make
   cd ../web-api
   make
+  cd ../core
+  make html
```

Comment:
Can you add a TODO here to use plain make?

Reply:
Switched to plain make.